packages/jsii-pacmak/test/expected.jsii-calc/dotnet/Amazon.JSII.Tests.CalculatorPackageId/Amazon/JSII/Tests/CalculatorNamespace/PartiallyInitializedThisConsumerProxy.cs

```diff
@@ -0,0 +1,18 @@
+using Amazon.JSII.Runtime.Deputy;
+
+namespace Amazon.JSII.Tests.CalculatorNamespace
+{
+    [JsiiTypeProxy(typeof(PartiallyInitializedThisConsumer), "jsii-calc.PartiallyInitializedThisConsumer")]
```

Comment:
If this is an interface that customers are actually supposed to implement, and not simply a pointer to implementations that were not exported, then proxies should not be generated for this interface. We might need to add an extra JSII flag to denote this difference.

Reply:
That's an abstract class, not an interface, actually. And I don't see why the JS-land couldn't send an instance of that out to the .NET land, so the .NET land would still need the proxy/deputy.